README.md

```diff
@@ -18,7 +18,7 @@ under an Apache 2.0 open source license.
 #### Requirements
 
 * Softimage 2015 SP1
-* Arnold 5.0.1.1 or newer
```

Comment:
Not really a requirement

Reply:
But Arnold itself is a requirement, so it needs to be in this section somehow. We might as well take the last version to avoid people compiling with 5.0.0.0 and hitting bugs that were already fixed.